Orange/widgets/utils/domaineditor.py

```diff
@@ -0,0 +1,186 @@
+from PyQt4 import QtCore, QtGui
+from PyQt4.QtGui import QTableView, QSizePolicy
+
+from Orange.data import DiscreteVariable, ContinuousVariable, StringVariable
+from Orange.widgets import gui
+from Orange.widgets.data.owcolor import HorizontalGridDelegate
+from Orange.widgets.utils.itemmodels import TableModel
+
+
+class Column:
+    name = 0
+    tpe = 1
+    place = 2
+    values = 3
+    not_valid = 4
+
+
+class Place:
+    feature = 0
+    class_var = 1
+    meta = 2
+    skip = 3
+
+
+class VarTableModel(QtCore.QAbstractTableModel):
+    places = "feature", "class", "meta", "skip"
+    typenames = "nominal", "numeric", "string"
+    vartypes = DiscreteVariable, ContinuousVariable, StringVariable
```

Comment:
We should probably include TimeVariable too.
With "time" or "datetime" typename maybe.

Confer with @kernc if more input is needed about this.